ddocs/medic-client/views/reports_by_form_year_month_places/map.js

```diff
@@ -1,71 +1,24 @@
 // data record must adhere to property name of `month`
 function(doc) {
-  if (doc.type !== 'data_record') {
-    return;
+  if (doc.type === 'data_record' &&
```

Comment:
Yeah I generally go for the return early principle too. I found this one confusing because it was split into two `if` statements and it was overly negative, ie: `a && b && c` is easier to read than `!a || !b || !c`.